Improving the Security concepts overviews (#2224)
dapr/docs: daprdocs/content/en/operations/components/component-scopes.md

{{< /tabs >}}

> Note: It is possible to have the component YAML applied to namespace "A" *reference* the implementation in namespace "B". For example, a component YAML for Redis in namespace "production-A" can set the Redis host address to point to the Redis instance deployed in namespace "production-B". See [Configure Pub/Sub components with multiple namespaces]({{< ref "pubsub-namespaces.md" >}}) for an example.

Reviewer: ```suggestion
{{% alert title="Note" color="primary" %}}
The component YAML applied to namespace "A" can *reference* the implementation in namespace "B". For example, a component YAML for Redis in namespace "production-A" can point the Redis host address to the Redis instance deployed in namespace "production-B". 

See [Configure Pub/Sub components with multiple namespaces]({{< ref "pubsub-namespaces.md" >}}) for an example.
{{% /alert %}}
```

## Application access to components with scopes

> Developers and operators might want to limit access for one database to a certain application, or a specific set of applications.

Reviewer: ```suggestion
Developers and operators might want to limit access to one database from a certain application, or a specific set of applications.
```

> To achieve this, Dapr allows you to specify `scopes` on the component YAML. Application scopes added to a component limit only the applications with specific IDs to be able to use the component.

Reviewer: ```suggestion
To achieve this, Dapr allows you to specify `scopes` on the component YAML. Application scopes added to a component limit only the applications with specific IDs from using the component.
```
